### PR TITLE
Add a Rust course

### PR DESCRIPTION
**Pull request by bruceadams**
Addresses #492

### CODE REVIEW

File: app/models/course.rb [hunk 1]
```diff
@@ -493,6 +493,24 @@ class Course < ActiveHash::Base
           ]
         }
       ]
+    }, {
+      id: 13,
+      name: "RUST",
+      title: 'Rust',
+      description: 'This is a Rust event. The focus will be on programming in Rust.',
```
Comment (ultrasaurus) [inline app/models/course.rb]: Suggest title: 'Rust for programmers'
description: 'Some programming knowledge expected, but you don't yet know Rust'

**Comment by ultrasaurus** [comment]
@bruceadams since most bridge workshops include options for non-programmers, I suggested a change to the title and description to make that clear.

In the future, we might have other Rust workshops for a broader audience.

File: app/models/course.rb (hunk 1 above)
Comment (bruceadams) [inline app/models/course.rb]: Yes! Your proposed changes are a big improvement. I've added a commit with these changes for Rust.

I'd like to go back and make similar improvements the title and description for Clojure. (I helped with ClojureBridge Pittsburgh a few weeks ago.) I'll do that as a separate pull request.

The title and text for Elm is also very similar. I'll see if I can track down those folks and suggest enhancing the text there as well.

File: app/models/course.rb [hunk 2]
```diff
@@ -493,6 +493,24 @@ class Course < ActiveHash::Base
...
+      title: 'Rust for programmers',
+      description: 'This is a Rust programming event. Some programming knowledge expected, but you don\'t yet know Rust.',
+      levels: [
+        {
+          level: 4,
+          color: 'orange',
+          title: 'Other Programming Experience',
+          level_description: [
+            'You\'re proficient in another language and understand general programming
+concepts, like collections and scope.',
```
Comment (bruceadams) [inline app/models/course.rb]: Oops! Looks like there is a line break in here. Not what I intended…

Comment (bruceadams) [inline app/models/course.rb]: Hmm. Elm and Clojure have the same line break. I guess I'll leave it as is.

**Comment by melancholyfleur** [comment]
Thanks again for doing this @bruceadams ^_^ 

I should say that it does definitely address a need for #492 but we will still need our chapter added, which it looks like that will require an actual DB migration. I can do that in a separate PR.

**Pull request merged**